smartsim/experiment.py

```diff
@@ -158,7 +157,7 @@ def __init__(self, name: str, exp_path: str | None = None):
         experiment
         """
 
-    def start(self, *jobs: Job | t.Sequence[Job]) -> tuple[LaunchedJobID, ...]:
+    def start(self, *jobs: Job | t.Sequence[Job]) -> tuple[Record, ...]:
         """Execute a collection of `Job` instances.
```

Comment:
It looks like there is a :raises TypeError: but I do not see a TypeError raised in this function?

Reply:
Whoops! I think that check got moved into `unpack` and we forgot to update the docstrings. I can move it there rq!